lib/DDGC/Web/Controller/InstantAnswer.pm

```diff
@@ -805,6 +805,12 @@ sub save_edit :Chained('base') :PathPart('save') :Args(0) {
         }
     }
 
+    $c->ddgcr_post( [ 'ActivityFeed', 'new' ], {
```

Comment:
Yeah, having this in the controller makes no sense anyway, since we can do Moo* hooks in Result classes to generate activity (same way notifications work now).

Cron jobs can also `use DDGC::Web::Base::Common;` and have access to the same facilities as other applications, including `rset`.

[`ddgcr_get/post`](https://github.com/duckduckgo/community-platform/blob/c2a62a2/lib/Dancer2/Plugin/DDGC/Request.pm) themselves could probably use improvement. If we have to abstract the abstraction, the abstraction is broken. Luckily there's not too much invested in it.

In this instance, anything in DDGC::* should (ab)use the Schema so these calls are going away.